tests/acceptance/browzine-primo-adapter-test.js

```diff
@@ -4125,18 +4143,121 @@ describe("BrowZine Primo Adapter >", function() {
 
       primo.searchResult($scope);
 
-      var request = jasmine.Ajax.requests.mostRecent();
-      expect(request).toBeUndefined();
+      var thirdIronApiDoiRequest = jasmine.Ajax.requests.mostRecent();
+
+      thirdIronApiDoiRequest.respondWith({
+        status: 200,
+        response: JSON.stringify({
+          "data": {
+            "id": 572672990,
+            "type": "articles",
+            "title": "The Residents of The Comoros and Sustainable Tourism",
+            "date": "2023-06-02",
+            "authors": "Sarı, Ömer; Meydan Uygur, Selma; Abdourahmane, Ali",
+            "inPress": false,
+            "abandoned": false,
+            "doi": "10.1163/09744061-bja10082",
+            "linkResolverOpenUrl": "",
+            "pmid": "",
+            "openAccess": false,
+            "unpaywallUsable": true,
```

Comment:
_non-blocking PR comment, no action required_

It strikes me as kind of strange that we have an `unpaywallUsable` value that could very reasonably be true in the API, and we also have `avoidUnpaywallPublisherLinks` ... from what I can tell: 

* `unpaywallUsable` set to true means that we are not actively suppressing Unpaywall data
* `avoidUnpaywallPublisherLinks` means we should not make live unpaywall calls and use the publisher links

So I understand why they each exist, I guess it's just a little odd to me that they could both be at odds and they seem very... similarly named 😅

Reply:
Yeah. It is odd. My understanding is that ```unpaywallUsable``` means a library opts to use unpaywall data if an article  isn't available, whereas ```avoidUnpaywallPublisherLinks``` is very specifically saying if we have a bad publisher link from unpaywall then don't use the unpaywalldata. So yes, a bit confusingly named, but they are distinct I think.

And this example is one I pulled from the API call I did while manually testing, so yeah. The way I read it is "use unpaywall...but not if they have bad publisher links"

Reply:
Thanks for clarifying that @Evroble and @mike-lang ! I edited the comment, now the definition of `avoidUnpaywallPublisherLinks` is correct after today's meeting.

Just some of the complexity of our system and making the user experience smooth for our users 😊